package.json

```diff
@@ -31,9 +30,12 @@
     "uglify-js": "^2.4.16",
     "watchify": "^2.2.1"
   },
+  "peerDependencies": {
+    "coffeeify": "^1.0.0"
```

Comment:
adding `  --ignore-transform coffeeify \ ` to the watchify command in bin/serve.sh seems to fix it